auth_otp/settings_sample.py

```
@@ -0,0 +1,216 @@
+SIMPLE_JWT = {
+    # زمان اعتبار توکن‌ها
+    'ACCESS_TOKEN_LIFETIME': timedelta(minutes=5),
+    'REFRESH_TOKEN_LIFETIME': timedelta(days=7),
+    
+    # Rotation و Blacklist
+    'ROTATE_REFRESH_TOKENS': True,
+    'BLACKLIST_AFTER_ROTATION': True,
+    
+    # الگوریتم و کلیدها
+    'ALGORITHM': 'HS256',
+    'SIGNING_KEY': SECRET_KEY,
```

Ruff (0.12.2)

Undefined name `SECRET_KEY`

(F821)